Loenn/entities/misc/railed_move_block.lua

```diff
@@ -35,11 +35,14 @@ for i, steeringMode in ipairs(steeringModes) do
             width = 16,
             height = 16,
             steeringMode = steeringMode,
-            speed = 120.0
+            speed = 120.0,
+			easedVerticalMovement = false,
+			attachedAbove = false
```

Comment:
```suggestion
            speed = 120.0,
            easedVerticalMovement = false,
            attachedAbove = false
```

Tiny nit, we use spaces to indent

Reply:
fixed.